src/connector.rs

```diff
@@ -0,0 +1,92 @@
+//! Utility methods for instantiating common connectors for clients.
```

Comment:
How does this relate to https://gitlab.datcon.co.uk/orchestration/hyper-builders?